src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -129,6 +129,7 @@ internal class DaprClientGrpc : DaprClient
             if (content != null)
             {
                 envelope.Data = content;
+                envelope.DataContentType = "application/json";
```

Comment:
test? SInce this is now something that the Dapr runtime cares about it should be in our tests.